backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5712360_sys_gh16844_Add_IsPostalAddress_Partner_Location.sql

```diff
@@ -0,0 +1,61 @@
+-- 2023-12-01T13:54:50.587Z
+INSERT INTO AD_Element (AD_Client_ID,AD_Element_ID,AD_Org_ID,ColumnName,Created,CreatedBy,EntityType,IsActive,Name,PrintName,Updated,UpdatedBy) VALUES (0,582828,0,'IsPostalAddress',TO_TIMESTAMP('2023-12-01 14:54:49','YYYY-MM-DD HH24:MI:SS'),100,'D','Y','Postadresse','Postadresse',TO_TIMESTAMP('2023-12-01 14:54:49','YYYY-MM-DD HH24:MI:SS'),100)
```

Comment:
This does not belong in metasfresh repo.
Is a column added for this customer.

Reply:
- There is already a customer windows (540905 - standard is 123)
- IMHO `IsShipTo` or `IsHandoverLocation` has a meaning very far away from `IsPostalAddress` is 
- Ok i will move it to customer repo

Reply:
The org partner are maintained from the Dist Org window and not from regular partner window.
This is the correct window ID: 540676